### PR TITLE
Update sklmessages_pl_PL.properties

### PR DESCRIPTION
**Pull request by Waterrail**

### CODE REVIEW

File: assets/lang/sklmessages_pl_PL.properties [hunk 1]
```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
...
-editor.alpha=Alfa
+editor.alpha=Alpha
 editor.snap=Snapshoty
-editor.version=U\u017cyj wersji
+editor.version=U\u017cyj wersji:
 editor.latest=U\u017cywaj najnowszej wersji
-editor.exe=Executable
-editor.jvm=JVM Arguments
-editor.copy=Kopia
+editor.exe=Plik wykonywalny:
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Labels shouldn't end with `:`
https://material.io/guidelines/components/text-fields.html#text-fields-layout

File: assets/lang/sklmessages_pl_PL.properties [hunk 2]
```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
...
+editor.visibility=Widoczno\u015b\u0107 launchera
 editor.resolution=Rozdzielczo\u015b\u0107
-editor.gamedir=Folder Gry
+editor.gamedir=Folder gry
 editor.beta=Beta
-editor.alpha=Alfa
+editor.alpha=Alpha
 editor.snap=Snapshoty
-editor.version=U\u017cyj wersji
+editor.version=U\u017cyj wersji:
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Labels shouldn't end with `:`
https://material.io/guidelines/components/text-fields.html#text-fields-layout

File: assets/lang/sklmessages_pl_PL.properties [hunk 3]
```diff
@@ -1,26 +1,26 @@
 # Contributors History:
 # skmedix: 24.08.2017 - ????
 
-lang.name=Polski
+lang.name=polski
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Language names should be capitalized to match other langs in combo box

File: assets/lang/sklmessages_pl_PL.properties [hunk 4]
```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
...
+editor.alpha=Alpha
 editor.snap=Snapshoty
-editor.version=U\u017cyj wersji
+editor.version=U\u017cyj wersji:
 editor.latest=U\u017cywaj najnowszej wersji
-editor.exe=Executable
-editor.jvm=JVM Arguments
-editor.copy=Kopia
+editor.exe=Plik wykonywalny:
+editor.jvm=Argumenty JVM:
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Labels shouldn't end with `:`
https://material.io/guidelines/components/text-fields.html#text-fields-layout

File: assets/lang/sklmessages_pl_PL.properties [hunk 5]
```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
...
+editor.copy=Kopia:
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Profile names with `:` looks a little bit weird

File: assets/lang/sklmessages_pl_PL.properties [hunk 6]
```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
...
-settings.token=Special Access Token
+settings.token=Specjalny klucz dostępu
 settings.save=Zapisz
 settings.cancel=Anuluj
 settings.info=Informacja
-settings.confirm=Musisz zrestartowa\u0107 launcher by zastosowa\u0107 zmiany
-settings.theme=Motyw
\ No newline at end of file
+settings.confirm=Musisz zrestartowa\u0107 launcher, aby zastosowa\u0107 zmiany
+settings.theme=Motyw
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: [Add missing crash reports strings please ](https://github.com/skmedix/SKlauncher/blob/master/assets/lang/sklmessages_en_US.properties#L67-L69)

File: assets/lang/sklmessages_pl_PL.properties [hunk 7]
```diff
@@ -1,26 +1,26 @@
...
-lang.name=Polski
+lang.name=polski
 lang.code=pl_PL
 
 tab.news=Aktualno\u015bci
-tab.log=Logi Launchera
-tab.output=Logi Gry (%s)
-tab.crash=Raport Crasha (%s)
+tab.log=Dziennik launchera
+tab.output=Dziennik Gry (%s)
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: The second word should also be lowercase

Comment (Waterrail) [reply assets/lang/sklmessages_pl_PL.properties]: Oh I forgot

File: assets/lang/sklmessages_pl_PL.properties [hunk 8]
```diff
@@ -1,26 +1,26 @@
 # Contributors History:
 # skmedix: 24.08.2017 - ????
```
Comment (skmedix) [inline assets/lang/sklmessages_pl_PL.properties]: Also you can add yourself in contributors history

File: assets/lang/sklmessages_pl_PL.properties (hunk 5 above)
Comment (Waterrail) [reply assets/lang/sklmessages_pl_PL.properties]: After a colon, you can use words in the nominative case. If wasn't included, then you would get things like "Kopia mój profil" etc.

**Comment by skmedix** [comment]
LGTM, thanks!

**Pull request merged**